config/i18n.json

```diff
@@ -487,6 +487,7 @@
   "Loadouts": {
     "Abandon": "Are you sure you want to abandon this loadout?",
     "AddEquippedItems": "Add Equipped",
+    "AddInventoryItems": "Add Inventory Items",
```

Comment:
Maybe call this "Add Unequipped"? 

Also, run `yarn i18n` to regenerate strings to include your edit.

Reply:
This makes total sense. Changed the names. Thank you!